src/Makefile

```diff
@@ -10,15 +10,12 @@ else
 XOBJS = $(DOBJS)
 endif
 
-CFLAGS-add += -DMATHLIB_STANDALONE
-
-
 default: release
 
 %.o: %.c
-	$(CC) $(CPPFLAGS) $(CFLAGS-add) $(CFLAGS) $(fPIC) -std=gnu99 -I../include -DNDEBUG -c $< -o $@
+	$(CC) $(CPPFLAGS) $(CFLAGS) $(CFLAGS-add) $(fPIC) -DMATHLIB_STANDALONE -I../include -DNDEBUG -c $< -o $@
```

Comment:
I think you'll still have to specify C99 somewhere in here; the tests are failing because it isn't assuming C99.